source/templates/committer-welcome.txt

```diff
@@ -0,0 +1,67 @@
+To: private@[PROJECT].a.o, NewCommitterID@apache.org
+Subject: Welcome, New Committer!
+
+[Committer Name] as you know, the ASF Infrastructure has set up your
```

Comment:
```suggestion
Hello [Committer Name],

As you know, the ASF Infrastructure has set up your
```

This is quite a bit better and well estabilished ettiquete to have greetings and person's name in a separate line. It also looks better this way.